scripts/lint_yaml.py

```diff
@@ -1,14 +1,22 @@
 #!/usr/bin/env python
-import re
-import os
-import sys
+from dataclasses import asdict, dataclass, field
 import datetime
 import glob
+import os
+import re
+import sys
+from typing import Dict, List, Optional
+
 import click
 from openstates import metadata
 from enum import Enum, auto
+from retire import (
```

Comment:
> Thanks for this, a lot here, going to add some inline thoughts, but also a couple of questions.
> 
> 1. is there a reason to have lint_yaml modify files, that seems a bit dangerous/unintuitive when retire.py could be modified to do the bulk retirement as needed?
> 2. can you explain data_root?
> 3. what is the logic with linting on inconsistent file names, the file name isn't an important part of the linting process by design, but curious to hear your thinking here

Item 3 first because easy...the rule is basically "if you tokenize `name` and `family_name` and ignore all non-english characters, there has to be at least one matching token between the two"

https://github.com/resistbot/people/blob/2b07015f3105d8f19e4f655d191c25ddd04832b1/scripts/lint_yaml.py#L401